tests/ui/methods/method-self-arg-trait.rs

```diff
@@ -64,4 +65,5 @@ fn main() {
     x.baz();
 
     unsafe { assert_eq!(COUNT, 2*2*3*3*5*5*7*7*11*11*13*13*17); }
+    //~^ WARN creating a shared reference to mutable static is discouraged [static_mut_refs]
```

Comment:
Since this test isn't related to references to mutable statics, we probably shouldn't warn here.  We could, e.g., copy `COUNT` to a local variable and then use that in the `assert_eq!`.

Reply:
How?

Reply:
E.g.:

```rust
let count = unsafe { COUNT };
assert_eq!(count, 2*2*3*3*5*5*7*7*11*11*13*13*17);
```